### DISCUSSION TITLE
请求彻底关闭键政话题

### DISCUSSION THREAD

**Discussion by QqhaZxAtbwhhk2Gl** [HyNetworks/hysteria#1237] [General]
尊敬的Toby和其他管理员：

您好！感谢您对本群和项目的持续投入和管理。近期，我注意到群内“键政”话题的讨论频繁且较为激烈，这不仅在浏览和加载其他话题时造成了一定的卡顿，还可能增加一些成员的潜在风险。此外，这类话题的讨论似乎对项目的发展并没有直接帮助，反而分散了成员们对核心内容的关注。

为此，我建议考虑关闭该话题的讨论或限制其频率，以便群内的交流更为聚焦，进一步推动项目的正向发展。

感谢您的理解和辛勤管理！

祝hysteria2蒸蒸日上！

**Comment by SakuraRK** [comment]
正确的
见证只会让这些灰色地带更容易被查到

**Comment by rg216** [comment]
主要是在飞机群里总会有人jz，但是如果没有这个板块的话，jz人就会跑到其他板块去污染讨论组？

**Comment by pryty26** [comment]
管理员为什么没有回复呢

**Comment by tobyxdd** [comment]
单独的建政话题目前已经关闭，相关讨论可转移至闲聊话题。

感谢您的反馈与建议。不过允许讨论此类讨论是本群一直以来的管理政策，目前没有调整的计划。如果您对这类内容感到不适，可以选择退群并仅关注我们的官方发布频道 (https://t.me/hysteria_releases) 此频道不会发布任何项目以外的内容。感谢理解。
